src/utilities/filterMovieAttributes.ts

```diff
@@ -0,0 +1,22 @@
+export default (data: any[], filters: string) => {
+    let requestedKeys = filters
+        .split(',')
+        .map(field => field.trim())
+        .filter(Boolean);
+
+    if ( requestedKeys.length ) {
+        data = data
+            .map((item: any) => {
+                for (const key in item) {
+                    if ( !requestedKeys.includes(key) ) {
+                        delete item[key];
```

Comment:
i would avoid delete operator construct the object from the requestedKeys

Reply:
Reworked.